app/controllers/api/messages_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Api::MessagesController < ApplicationController
```

Comment:
Okay, here's a new API controller for sending messages. It's basically identical to what I was doing with the previous messages_controller#create action, but with the added bonus of "I can post to it with ajax".